function-images/tests/chained-function-serving/producer/producer.go

```diff
@@ -77,6 +112,7 @@ func main() {
 	flagClientPort := flag.Int("pc", 80, "Client Port")
 	flagServerPort := flag.Int("ps", 80, "Server Port")
 	url := flag.String("zipkin", "http://zipkin.istio-system.svc.cluster.local:9411/api/v2/spans", "zipkin url")
+	transferSize := flag.Int("transferSize", 4095, "Number of KB's to transfer")
```

Comment:
why not 4096? that's weird

Reply:
it exceeds the gRPC limit by a few bytes.